src/pages/Inventory.page.js

```diff
@@ -40,4 +46,18 @@ export class InventoryPage extends BaseSwagLabPage {
 
         return cardNames.map((name) => name.toLowerCase());
     }
+
+    async getInventoryItemsDescriptions() {
+        const cardDescriptions = await this.page.locator(this.inventoryItemsDescription).allTextContents();
+
+        return cardDescriptions.map((description) => description.toLowerCase());
```

Comment:
adding toLowerCase changes descriptions we get. In case I call getInventoryItemsDescriptions and get results in lower case I can check what I see on UI (not the lower case) and this is called side effect. Nobody knows about this 2lowerCase except you :) Sooo, to fix this you could rename this function getInventoryItemsDescriptionsInLowerCase or remove this function and use this code as is.